Misc/NEWS.d/next/Library/2018-10-27-21-11-42.bpo-34160.UzyPZf.rst

```diff
@@ -1 +1,2 @@
-ElementTree and minidom now preserve the attribute order specified by the user.
+Added support for the *sort_attrs* argument in methods for serializing XML in
+modules :mod:`xml.etree.ElementTree` and :mod:`xml.dom.minidom`.
```

Comment:
No, the default was not changed. I think it is better first add an option, and change the default in next the version. This will make supporting two consequent versions (3.7-3.8 or 3.8-3.9) easier.